src/targets/gpu/CMakeLists.txt

```diff
@@ -239,9 +239,14 @@ endif()
 include(CheckLibraryExists)
 get_target_property(MIOPEN_LOCATION MIOpen LOCATION)
 check_library_exists(MIOpen "miopenHiddenSetConvolutionFindMode" "${MIOPEN_LOCATION}" HAS_FIND_MODE_API)
-if(HAS_FIND_MODE_API)
+check_library_exists(MIOpen "miopenFindSolutions" "${MIOPEN_LOCATION}" HAS_FIND_2_API)
+
+if(HAS_FIND_2_API) 
+    target_compile_definitions(migraphx_gpu PUBLIC -DMIGRAPHX_HAS_FIND_2_API)
+    message(STATUS "MIGraphx is using Find-2.0 API of MIOpen")
+elseif(HAS_FIND_MODE_API)
```

Comment:
So we still need this find mode API with find 2.0 now because MIOpen doesn't provide an API for it yet. We could address this with a later PR.

Reply:
I think changing from `elseif` to `if` would fix this, since this is called on the convolution descriptor which find 2.0 will still use.